runWPTAll.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# shellcheck disable=SC2155
+
+set -uo pipefail
+
+trap ctrl_c INT
+ctrl_c() {
+    exit 1
+}
+
+export LOG_FILE="logs/$(basename "$0").log"
+export UPDATE_EXPECTATIONS="${UPDATE_EXPECTATIONS:-false}"
```

Comment:
I have needed to use it yet so, @jrandolf do you think setting update expectation default to true is more in line with what this script will be used for?

Reply:
@thiagowfx

Reply:
I tend to use both values. `false` during development, `true` when I am ready to regenerate the expectations. We can set this to `false`.